WindowsServerDocs/storage/storage-spaces/data-collection.md

```diff
@@ -29,12 +29,15 @@ There are two methods of installing the script, both of which are outlines below
 
 The [PowerShell Gallery](https://www.powershellgallery.com/packages/PrivateCloud.DiagnosticInfo) is a snapshot of the GitHub Repo. Note that installing items from the PowerShell Gallery requires the latest version of the PowerShellGet module, which is available in Windows 10, in Windows Management Framework (WMF) 5.0, or in the MSI-based installer (for PowerShell 3 and 4).
 
+We install the latest version of the [Microsoft Networking Diagnostics tools](https://www.powershellgallery.com/packages/MSFT.Network.Diag) during this process as well since Get-SDDCDiagnosticInfo relies on this. This manifest module contains network diagnostic and troubleshooting tool, which are maintained by the Microsoft Core Networking Product Group at Microsoft.
```

Comment:
Grammar/fluency/natural speech improvement suggestion:
```suggestion
During this process, we will install the latest version of the [Microsoft Networking Diagnostics tools](https://www.powershellgallery.com/packages/MSFT.Network.Diag) as well, since Get-SDDCDiagnosticInfo relies on this. This manifest module contains network diagnostic and troubleshooting tools, which are maintained by the Microsoft Core Networking Product Group at Microsoft.
```
(Not really sure if the 2 last words "at Microsoft" are strictly needed, but I am neither an MS internal nor a native English speaker. I will leave that decision up to either a copy review contributor or one of the MS team members.)

Reply:
@adagashe Please remove at Microsoft as @illfated suggested and I will merge this PR.  Thanks for the feedback!

Reply:
Thank you for confirming that part. I will modify my suggestion so it can be merged with the rest of my suggestion.

Reply:
Modification done. My suggestion is ready to commit.